docs/en/engines/database-engines/materialize-mysql.md

```diff
@@ -7,7 +7,7 @@ toc_title: MaterializeMySQL
 
 Creates ClickHouse database with all the tables existing in MySQL, and all the data in those tables.
 
-ClickHouse server works as MySQL replica. It reads binlog and performs DDL and DML queries.
+ClickHouse server works as MySQL replica. It reads binlog and performs DDL and DML queries. ClickHouse database created with MaterializeMySQL contains all column comments from the MySQL database that materialized.
```

Comment:
This info about column comments is not as important to be in this section. Enough to add to `### Selecting from MaterializeMySQL Tables {#select}`.

Reply:
Ok, i'll remove it ;)